src/ansys/geometry/core/geometry/parameterization.py

```diff
@@ -202,6 +231,139 @@ def get_span(self) -> Real:
 
         return self.end - self.start
 
+    def get_relative_val(self, t: Real) -> Real:
+        """
+        Return an evaluation property of the interval, used in BoxUV.
+
+        Parameters
+        ----------
+        t : Real
+            The offset that the interval gets evaluated at.
+
+        Returns
+        -------
+        Real
+            The actual value according to the offset
+        """
+        return self.start + t * self.get_span()
+
+    def is_negative(self, tolerance: Real) -> bool:
+        """
+        Boolean value that indicates whether the current interval is negative.
+
+        Parameters
+        ----------
+        tolerance : Real
+            The accepted range since we could be working with doubles
+
+        Returns
+        -------
+        bool
+            True if negative False otherwise
```

Comment:
```suggestion
        tolerance : Real
            Accepted range because the data type of the interval could be in doubles.

        Returns
        -------
        bool
            ``True`` if the interval is negative, ``False`` otherwise.
```